src/components/views/symptom-checker/symptoms.tsx

```diff
@@ -54,13 +54,20 @@ export const CheckInSymptoms = () => {
   });
 
   const [symptoms, setSymptoms] = useState<SymptomRecord>(values);
+  const [saving, setSaving] = useState(false);
 
   const gotoResults = () => {
+    if (saving) {
+      return;
+    }
+    setSaving(true);
     try {
       app.checkIn(symptoms, {feelingWell: false});
     } catch (err) {
       console.log('Check-in error', err);
+      setSaving(false);
```

Comment:
Why do we need to set false here?

Reply:
Oh yeah, I missed that we continue with the navigation even if there was an error. So it's uneccessary.